rest/src/main/java/discord4j/rest/util/InteractionResponseType.java

```diff
@@ -58,6 +58,7 @@ public enum InteractionResponseType {
     /**
      * Response to a supported interaction with a notification that this interaction is only available to premium guilds or users.
      */
+    @Deprecated
     PREMIUM_REQUIRED(10);
```

Comment:
Can be good add in docs the deprecated reason based in docs https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-response-object